lib/nodes/fpga.cpp

```diff
@@ -106,9 +119,10 @@ int FpgaNode::parse(json_t *json) {
     vfioContainer = std::make_shared<kernel::vfio::Container>();
   }
 
-  ret = json_unpack_ex(json, &err, 0, "{ s: o, s?: o, s?: b, s?: b}", "card",
-                       &jsonCard, "connect", &jsonConnectStrings,
-                       "lowLatencyMode", &lowLatencyMode);
+  ret =
+      json_unpack_ex(json, &err, 0, "{ s: o, s?: o, s?: b, s?: b, s?: f}",
+                     "card", &jsonCard, "connect", &jsonConnectStrings,
+                     "lowLatencyMode", &lowLatencyMode, "timestep", &timestep);
```

Comment:
Can we please use snake case for option names? Camel case isnt consistent with the other settings we have in VILLASnode.

Reply:
this I did not know. I changed it.